Makefile

```diff
@@ -0,0 +1,75 @@
+# Makefile includes some useful commands to build or format incentives
+# More commands could be added
+
+# Variables
+PROJECT = gouberfmt
+REPO_ROOT = github.com/gophersbd
+DOCKER_PROJECT = sadlil
```

Comment:
Something generic would be nice? :3

Reply:
Ah. I forgot to remove this. after testing. will remove this. Thanks. Nice catch.